src/api/libcellml/units.h

```diff
@@ -428,41 +428,41 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
     /**
      * @brief Return the scaling factor difference between two @c Units.
      *
-     * This can be interpreted as factor, where units2 = (factor)*units1.  This method
-     * does not check to see if the units are compatible.
+     * This can be interpreted as factor, where units2 = (factor)*units1.  If the units
+     * are not compatible the factor returned is 0.
      *
      * @param units1 The first units to compare.
      * @param units2 The second units to compare.
      *
-     * @return The factor units1/units2.
+     * @return The factor units2/units1, where the units are compatible, or 0.0 otherwise.
      */
     static double scalingFactor(const UnitsPtr &units1, const UnitsPtr &units2);
 
     /**
-     * @brief Test to determine whether two @c Units are equivalent or not.
+     * @brief Test to determine whether two @c Units are compatible or not.
      *
-     * Two @c Units are considered to be equivalent if they share the same units,
+     * Two @c Units are considered to be compatible if they share the same units,
      * independently of their dimension (e.g. volt and volt are equivalent as are
```

Comment:
I think this should change to independently of their scaling.  Dimension to me means we would consider volt and volt squared as being compatible.

Reply:
But ... that defeats the point of dimensional analysis? if you remove the dimension checking? I'm confused ...

Reply:
Ahhh ... I get it now, thanks.

Reply:
then dimensionality is a better term.

Reply:
Done.